src/macros/mod.rs

```diff
@@ -434,74 +437,313 @@ mod tests {
 
     #[test]
     fn test_rust_primitive_types_conversion_in_log() {
-        let (output, guard) = get_output_guard(Level::INFO);
+        let log_exporter = InMemoryLogExporter::default();
+
+        let logfire = crate::configure()
+            .local()
+            .send_to_logfire(false)
+            .with_advanced_options(
+                AdvancedOptions::default()
+                    .with_log_processor(SimpleLogProcessor::new(log_exporter.clone())),
+            )
+            .finish()
+            .unwrap();
+
+        let guard = crate::set_local_logfire(logfire);
 
         // i64::MAX =                9_223_372_036_854_775_807
         // so we test overflow with 10_000_000_000_000_000_000
 
-        crate::info!("an u8: {value}", value = 1u8);
-        crate::info!("an u16: {value}", value = 2u16);
-        crate::info!("an u32: {value}", value = 3u32);
-        crate::info!("an u64: {value}", value = 4u64);
+        crate::info!("u8: {value:?}", value = 1u8);
```

Comment:
Are `?` needed here?

Reply:
Probably not, no. Originally I thought this would show me the cases which overflow i64 as strings but of course when formatting the message the integer formats fine regardless of size. Will revert.

Reply:
Reverted (except on `char` and `Option` values).